fp-bindgen/src/generators/rust_plugin/mod.rs

```diff
@@ -62,12 +62,12 @@ fn generate_cargo_file(
                 git: Some("ssh://git@github.com/fiberplane/fp-bindgen.git"),
                 branch: Some("main"),
                 path: None,
-                version: None,
+                version: Some("1.0.0-rc.1"),
```

Comment:
We could replace this with `env!("CARGO_PKG_VERSION")` so it will use the same version.